### PR TITLE
ISPN-14761 Displays encoding message for Java and JBoss marshalling

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-14761

### CODE REVIEW

File: src/app/Caches/DetailCache.tsx [hunk 1]
```diff
@@ -109,6 +144,29 @@ const DetailCache = (props: { cacheName: string }) => {
     );
   };
 
+  const encodingMessage = () => {
```
Comment (dpanshug) [inline src/app/Caches/DetailCache.tsx:147]: @karesti This function is not used anywhere, is it anything we need in the future?

Comment (karesti) [reply src/app/Caches/DetailCache.tsx:147]: forgotten by mistake in the pr

File: src/app/assets/languages/en.json [hunk 2]
```diff
@@ -443,7 +443,8 @@
       "no-templates-body": "Add cache templates to the server configuration.",
       "table-label": "Create caches from custom templates.",
       "no-encoding-warning": "This cache either has no encoding configuration or uses an encoding that the console does not support. You must use Hot Rod clients to perform read and write operations on this cache.",
-      "no-encoding-docs": "See \"Cache Encoding and Marshalling\" in the documentation."
+      "encoding-docs-message": "See \"Cache Encoding and Marshalling\" in the documentation.",
+      "pojo-encoding": "This cache uses \"{{encoding}}\" encoding. Data manipulation or displaying entries might not be fully supported from the Console."
```
Comment (dvagnero) [inline src/app/assets/languages/en.json:447]: "{{brandname}} Console might not fully support data manipulation or displaying entries."
OR if I understand correctly 
"{{brandname}} Console might not fully support data manipulation or displaying entries for caches that don't use ProtoStream encoding."

**Comment by karesti** [comment]
@dvagnero I modified the message

**Review by dvagnero — APPROVED** [review]

File: src/app/assets/languages/en.json (hunk 2 above)
Comment (karesti) [reply src/app/assets/languages/en.json]: done

**Comment by karesti** [comment]
@dpanshug check this PR.
The caches without any encoding are the ones that have no encoding. Queriable is used to display the query tab or not in the code, nothing else

**Pull request merged**